File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.py
# Last updated: 14/07/2026, 14:32:14
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if not nums:
            return 0
        k=1
        for i in range(1,len(nums)):
            if nums[i]!=nums[k-1]:
                nums[k]=nums[i]
                k+=1
        return k
        

File: 26-RemoveDuplicatesFromSortedArray/26-RemoveDuplicatesFromSortedArray.java
// Last updated: 14/07/2026, 14:32:20
class Solution 
{
    public int removeDuplicates(int[] nums) 
    {
        int index=0;
        for(int i=0;i<nums.length;i++)
        {
            boolean flag=false;
            for(int j=0;j<i;j++)
            {
                if(nums[i]==nums[j])
                {
                    flag=true;
                }
            }
            if(!flag)
            {
                nums[index]=nums[i];
                index++;
            }
        }
        return index;
    }
}